oeps/conf.py

```diff
@@ -136,7 +136,6 @@
     "repository_url": "https://github.com/openedx/open-edx-proposals",
     "repository_branch": "master",
     "path_to_docs": "oeps",
-    "logo_only": False,
```

Comment:
`logo_only: Bool` field replaced with `logo_only: {}` option in the new version of `sphinx-book-theme`. For details see the change https://github.com/executablebooks/sphinx-book-theme/pull/668/files#diff-85933aa74a2d66c3e4dcdf7a9ad8397f5a7971080d34ef1108296a7c6b69e7e3L123